backend/src/main/resources/apis/api.yaml

```diff
@@ -0,0 +1,86 @@
+openapi: 3.0.0

```

Comment:
When I pasted this api contract into the online swagger editor it listed some errors and when I ran the frontend generator on the api it also listed some errors. So I think there are still some small things that need to be fixed with the contract